### DISCUSSION TITLE
npm package

### DISCUSSION THREAD

**Discussion by boomzero** [XMOJ-Script-dev/XMOJ-Script#208] [General]
https://www.npmjs.com/package/xmoj-bbs?activeTab=readme

**Comment by langningchen** [comment]
WTF? ...

**Comment by PythonSmall-Q** [comment]
wtf??

**Reply by langningchen** [reply]
@PythonSmall-Q 你怎么学我......

**Comment by boomzero** [comment]
怎么了

**Comment by boomzero** [comment]
我就npm publish了一下

**Comment by langningchen** [comment]
> 怎么了
> 
> > On Nov 12, 2023, at 11:37, Chen LangNing ***@***.***> wrote:
> > 
> > 
> > @PythonSmall-Q <https://github.com/PythonSmall-Q> 你怎么学我......
> > 
> > —
> > Reply to this email directly, view it on GitHub <https://github.com/orgs/XMOJ-Script-dev/discussions/208#discussioncomment-7543322>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AULMJZNKQVL7O6TA23AS5XTYEA76ZAVCNFSM6AAAAAA7HGGYSWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TKNBTGMZDE>.
> > You are receiving this because you authored the thread.
> > 
> 
> 

@boomzero 没怎么了
